src/lpspi.rs

```diff
@@ -9,20 +9,29 @@ pub enum Sck {}
 pub enum Sdo {}
 /// A tag that indicates a SPI data in pad
 pub enum Sdi {}
-/// A tag that indicates a SPI chip select pad
+/// Tags that indicate one of several SPI chip select pads
```

Comment:
Comments with three slashes are [documentation comments](https://doc.rust-lang.org/rust-by-example/meta/doc.html). They render in [the auto-generated docs](https://docs.rs/imxrt-iomuxc/0.2.1/imxrt_iomuxc/lpspi/enum.Pcs0.html). This new phrasing is helpful when browsing the source code. But, I'm not sure if will make sense when attached to the `Pcs0` type; `Pcs0` indicates one specific pad, not several pads.

I recommend adding dedicated comments to the new types. They can be duplicates of the original phrasing, or they can emphasize that they're specific to the zeroth / first / second / third chip select.

Reply:
Excellent, thank you! I've made the comments individual, and brought the new symbols into the namespace where they are needed. I've also made sure that clippy is satisfied, and all tests have passed.